tests/e2e/link_test.go

```diff
@@ -77,6 +85,13 @@ var _ = Describe("odoLinkE2e", func() {
 			Expect(output).To(ContainSubstring("been linked"))
 		})
 
+		It("describe should show the linked service", func() {
```

Comment:
Steps description looks like duplicate of line no 53 though it checks backend service. Please make both the sentences specific. For example 
```
It("describe should show the frontend linked service", func() {
[...]
}
```
```
It("describe should show the backend linked service", func() {
[...]
}
```

Reply:
Fixed